tests/Validators/SKUValidatorTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tests\Validators;
+
+use App\Validators\skuValidator;
+use Tests\TestCase;
+
+class skuValidatorTest extends TestCase
+{
+    public function testValidateSkuSuccess()
+    {
+        $sku = 'AAABBCCC11';
+        $result = skuValidator::validateSku($sku);
+        $this->assertEquals('AAABBCCC11', $result);
+    }
+
+    public function testValidateSKUFailure()
+    {
+        $sku = '@£$%';
+        $this->expectException(\Exception::class);     
```

Comment:
This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.